src/libstore/globals.hh

```diff
@@ -689,19 +690,38 @@ public:
         getDefaultSystemFeatures(),
         "system-features",
         R"(
-          A set of system “features” supported by this machine, e.g. `kvm`.
-          Derivations can express a dependency on such features through the
-          derivation attribute `requiredSystemFeatures`. For example, the
-          attribute
+          A set of system “features” supported by this machine.
+
+          This complements the [`system`](#conf-system) and [`extra-platforms`](#conf-extra-platforms) configuration options and the corresponding [`system`](@docroot@/language/derivations.md#attr-system) attribute on derivations.
+
+          Derivations can require system features in the derivation attribute [`requiredSystemFeatures`](@docroot@/language/advanced-attributes.md#adv-attr-requiredSystemFeatures).
+
+          System features are generally user-defined, but the following have special treatment:
```

Comment:
```suggestion
          While new system features can be defined freely, the default set of features is based on Nix's environment and configuration as follows:
```

I don't think the treatment is very special, except for `ca-derivations`, although that's on the derivation side.